<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvironmentSetupDialog</class>
 <widget class="QDialog" name="EnvironmentSetupDialog">
  <property name="windowTitle">
   <string>Environment Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="step1Label">
     <property name="text">
      <string>‣ Extract the driver to a folder:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="extractButton">
     <property name="text">
      <string>Select Folder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="step2Label">
     <property name="text">
      <string>‣ Open a terminal and run the following commands:</string>
     </property>
    </widget>
   </item>
   <item>
        <layout class="QHBoxLayout" name="commandLayout">
            <item>
            <widget class="QTextEdit" name="commandsTextEdit">
                <property name="readOnly">
                <bool>true</bool>
                </property>
                <property name="visible">
                <bool>false</bool>
                </property>
                <property name="styleSheet">
                <string>background-color:black; color:white;</string>
                </property>
            </widget>
            </item>
            <item>
            <widget class="QPushButton" name="copyButton">
            <property name="fixedWidth">
            <number>20</number>
            </property>
            <property name="text">
            <string>⎘</string>
            </property>
            </widget>
        </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 